docs(tutorials): microsoft hyper-v (#109)
scaleway/docs-content: tutorials/configure-vm-hyperv/index.mdx

>   Microsoft Hyper-V is a hypervizor which allows to create virtual machines on x86-64 systems. It is available since the release of Windows Server 2012.

Reviewer: ```suggestion
  Microsoft Hyper-V is a hypervizor which allows you to create virtual machines on x86-64 systems. It is available since the release of Windows Server 2012.
```

  A free stand-alone, but command-line only, version of Hyper-V Server is also available.
categories:
  - dedicated-servers
  - dedibox
image: /assets/images/docs/scaleway.png
keywords: cloud tutorials, cloud documentations, cloud instance, cloud hosting, windows server, hyper-v, windows hyper-v, virtual machine, windows
dates:

>   validation: 2020-10-28

Reviewer 2: ```suggestion
  validation: 2021-10-28
```

Your server can also be booted from a Windows Server ISO file through a KVM over IP device if included in your Dedibox. This type of installation gives you the possibility of installing your own licenses or remote ISO files.

<Message type="tip">

You can download Windows and HyperV ISO images from Microsoft’s [evaluation center](https://www.microsoft.com/en-us/evalcenter).

</Message>

1. Access the server page of your Dedibox on the console and click **Install** to begin installation via KVM over IP.

2. Click **on KVM over IP**. A Disclaimer window pops up. 

  <Lightbox src="scaleway-kvm-over-ip.png" width="600" alt="" />

> 3. Click **I ACCEPT** once you have read and agreed to the terms displayed. You are then redirected to your server's status page.

Reviewer 2: ```suggestion
3. Click **I ACCEPT** once you have read and agreed to the terms displayed. You are redirected to your server's status page.
```

  <Lightbox src="scaleway-disclaimer.png" width="300" alt="" />

> 4. Click the third button on the right. The model of the KVM over IP device is determined by the brand of your Dedibox chassis. For a Dell chassis, you will use **IDRAC**.

Reviewer 2: ```suggestion
4. Click the third button on the right. The model of the KVM over IP device is determined by the brand of your Dedibox chassis. For a Dell chassis, use **IDRAC**.
```

## Deploying a Virtual Machine

1. Click **New** in the **Actions** menu, then **Virtual Machine** in the Hyper-V Manager application to launch the virtual machine creation wizard:

  <Lightbox src="scaleway-hyperv_01.png" width="800" alt="" />

> 2. Click **Next** to begin with the configuration of the new virtual machine:

Reviewer 2: ```suggestion
2. Click **Next** to begin the configuration of the new virtual machine:
```

  <Lightbox src="scaleway-hyperv_07.png" width="600" alt="" />

> 8. Choose the installation options for the virtual machine. You can specify to create the machine without an operating system, specify a local ISO file or provide information for a network-based installation server. If you have downloaded an ISO file to install your OS, choose the second option and specify the file path before clicking **Next**. Then a configuration summary displays.

Reviewer: ```suggestion
8. Choose the installation options for the virtual machine. In order to create the machine without an operating system, specify a local ISO file or provide information for a network-based installation server. If you have downloaded an ISO file to install your OS, choose the second option and specify the file path before clicking **Next**. Then a configuration summary displays.
```

> 8. Choose the installation options for the virtual machine. You can either specify to create the machine without operating system, specify a local ISO file or provide information for a network based installation server. If you have downloaded an ISO file to install your OS, choose the second option and specify the file path before clicking **Next**. Then a configuration summary displays.

Reviewer 2: ```suggestion
8. Choose the installation options for the virtual machine. You can specify to create the machine without an operating system, specify a local ISO file or provide information for a network-based installation server. If you have downloaded an ISO file to install your OS, choose the second option and specify the file path before clicking **Next**. Then a configuration summary displays.
```

  <Lightbox src="scaleway-hyperv_08.png" width="600" alt="" />

9. Click **Finish** to launch the creation of the virtual machine:

  <Lightbox src="scaleway-hyperv_09.png" width="600" alt="" />

> 10. Assign a virtual MAC address to the failover IP from the [Dedibox Console](https://console.online.net/) to make sure that the virtual machine is able to communicate on the Internet. Refer to the [dedicated documentation](https://www.scaleway.com/en/docs/dedibox-network/ip-failover/how-to/create-virtual-mac/) for more information how to create a virtual MAC address. Click **Settings** in the VM section of the menu.

Reviewer 2: ```suggestion
10. Assign a virtual MAC address to the failover IP from the [Dedibox Console](https://console.online.net/) to make sure that the virtual machine is able to communicate on the Internet. Refer to the [dedicated documentation](https://www.scaleway.com/en/docs/dedibox-network/ip-failover/how-to/create-virtual-mac/) for more information about how to create a virtual MAC address. Click **Settings** in the VM section of the menu.
```

> 12. Start the Virtual Machine by clicking on **Start** in the side menu. The virtual machine will boot from the ISO file and the installtion wizard of the operating system displays. Follow the steps of the wizard to complete the installation.

Reviewer: ```suggestion
12. Start the Virtual Machine by clicking on **Start** in the side menu. The virtual machine will boot from the ISO file and the installation wizard of the operating system displays. Follow the steps of the wizard to complete the installation.
```
